src/ng/parse.js

```diff
@@ -1950,23 +1956,20 @@ function $ParseProvider() {
 
       var fn = parsedExpression.oneTime ? oneTimeInterceptedExpression : regularInterceptedExpression;
 
-      // Propogate the literal/oneTime attributes
+      // Propogate the literal/oneTime/constant attributes
       fn.literal = parsedExpression.literal;
       fn.oneTime = parsedExpression.oneTime;
+      fn.constant = parsedExpression.constant;
 
-      // Propagate or create inputs / $$watchDelegates
-      useInputs = !parsedExpression.inputs;
-      if (watchDelegate && watchDelegate !== inputsWatchDelegate) {
```

Comment:
Yeah they are just ignored, because the interceptor might have made that `$$watchDelegate` completely invalid. However the returned ("interecepted"?) fn might have some constant/oneTime/inputs properties that might cause a `$$watchDelegate` to be added.